src/targets/services/helpers/model.js

```diff
@@ -32,6 +34,11 @@ export class Model {
     return model
   }
 
+  static fromCompressedBackup(compressedBackup) {
```

Comment:
nit: the backup semantic bother me a little in this context, it sounds like this is used to save a copy of the model. Why not just  `fromCompressedModel`? I have the same feeling about `Model.fromBackup` that does not look like backup at all

Reply:
The `getBackup` function is used to get the minimal amount of data needed to reconstruct the model. It's then written to the disk. Also, it's not directly usable as a model